src/store/user/user-context.js

```diff
@@ -10,17 +10,30 @@ export const UserContextProvider = ({ children }) => {
   const [showMemberRoleUpdateModal, setShowMemberRoleUpdateModal] =
     useState(false);
   const [selectedMember, setSelectedMember] = useState(null);
+  const [showMemberTaskUpdateModal, setshowMemberTaskUpdateModal] =
+    useState(false);
```

Comment:
Can we have this in the same line i.e. 13 please ?

Reply:
will do that

Reply:
@rohan09-raj it is a problem with prettier, when I run yarn lint it changes it, and if I change it again it is not allowing me to commit